Docker.DotNet.BasicAuth/BasicAuthCredentials.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using System.Net.Http;
-#if !NETSTANDARD1_3
+#if (!NETSTANDARD1_3 && !NETSTANDARD1_6)
```

Comment:
Since this is going to be the more common case in the future, should we instead invert this to NET45 || NET46 ?